.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
.gitignore also should be ignored

Reply:
i read that, it should be done global, in console, is that correct?

Reply:
Yup, from the console. But actually there are two schools of thought on this subject so I leave it for you to decide if you want to include it.